docker/docker-compose.yml

```diff
@@ -24,9 +24,9 @@ services:
       kafka-zookeeper:
         condition: service_healthy
       pinot:
-        condition: service_healthy
+        condition: service_started
```

Comment:
why

Reply:
As I mentioned, to avoid the 60s limit of docker-compose on low configuration, and pinot sometimes starts on the edge, giving us an `un-healthy containers` issue.